schemas/enums/language.json

```diff
@@ -50,6 +50,7 @@
         "Docker",
         "Drools",
         "DTS",
+        "Earthly",
```

Comment:
I'd suggest using "Earthly" as the language entry. While technically not a language, these entries are primarily meant to allow users to discover relevant tooling and I believe Earthly would be the most appropriate one in this case.